simulator/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM rust:1.70.0-slim-bullseye as builder
+
+RUN apt update && apt install -y \
+    cmake=3.18.4-2+deb11u1 \
+    clang=1:11.0-51+nmu5 \
+    build-essential=12.9 \
+    llvm=1:11.0-51+nmu5
+#RUN rustup target add wasm32-unknown-unknown
+
+ADD simulator /app/simulator
+ADD radix-engine /app/radix-engine
+ADD radix-engine-interface /app/radix-engine-interface
+ADD sbor /app/sbor
+ADD sbor-derive /app/sbor-derive
+ADD sbor-derive-common /app/sbor-derive-common
+ADD utils app/utils
+ADD scrypto-schema /app/scrypto-schema
+ADD radix-engine-common /app/radix-engine-common
+ADD radix-engine-derive /app/radix-engine-derive
+ADD native-sdk /app/native-sdk
+ADD radix-engine-macros /app/radix-engine-macros
+ADD radix-engine-profiling /app/radix-engine-profiling
+ADD radix-engine-store-interface /app/radix-engine-store-interface
+ADD radix-engine-stores /app/radix-engine-stores
+ADD transaction /app/transaction
+ADD radix-engine-queries /app/radix-engine-queries
+ADD assets /app/assets
+
+WORKDIR /app
+
+RUN cargo install --path ./simulator
+
+FROM rust:1.70.0-slim-bullseye
```

Comment:
Do we even need to use the rust bullseye base? Or should we use the [corresponding Debian base](https://hub.docker.com/layers/library/rust/1.70.0-slim-bullseye/images/sha256-1de20372759e9341c5b6e8cc7b18239e91947cf5ba9ddc20c0e5903b2b9fde24?context=repo&tab=vulnerabilities): which is `bullseye-20230703-slim`? 

Happy with either tbh. I don't imagine the Rust image is much on top of bullseye slim, so maybe it's easier to be consistent.

Reply:
Oh hmm I guess we need the rust base image for the scrypto build anyway :D ignore me.

Reply:
right on spot :)